### PR TITLE
FIX: Apply value to derivative variable

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
## Description
Set tuning offset was not setting the value of the variable correctly

## Issue linked
#5497 

## Checklist
- [x] I have tested my changes locally.
- [x] I have added necessary documentation or updated existing documentation.
- [x] I have followed the coding style guidelines of this project.
- [x] I have added appropriate tests (unit, integration, system).
- [x] I have reviewed my changes before submitting this pull request.
- [x] I have linked the issue or issues that are solved by the PR if any.
- [x] I have agreed with the Contributor License Agreement ([CLA](https://developer.ansys.com/form/cla-acceptance)).

### CODE REVIEW

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 1]
```diff
@@ -2483,27 +2483,51 @@ def add_derivatives(self, derivative_list):
...
             if v not in offsets:
                 offsets[v] = 0
-        arg = []
+        arg = ["NAME:TuningRanges"]
         for k, v in offsets.items():
-            arg.append(f"DeltaOffset({k})")
-            arg.append(f"{abs(self._app.variable_manager[k].numeric_value) * (-0.1)}")
-            arg.append(f"{v}")
-            arg.append(f"{abs(self._app.variable_manager[k].numeric_value) * 0.1}")
+            arg.append(f"Range:=")
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/modules/solve_setup.py:2521]:
```suggestion
            arg.append("Range:=")
```
Only use f-string when there is a variable :)

File: src/ansys/aedt/core/modules/solve_setup.py [hunk 2]
```diff
@@ -3200,27 +3224,51 @@ def add_derivatives(self, derivative_list):
...
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/modules/solve_setup.py:3262]:
```suggestion
            arg.append("Range:=")
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left minor comments.
The two methods `set_tuning_offset` seem to be the exact same code. We should really think about refactoring a bit the code to avoid code duplicates.
Do we have an issue to track all those duplicates and see if we can't come up with new classes encapsulating all those duplicates ?

**Comment by Samuelopez-ansys** [comment]
I agree regarding the duplicities. In this case this method can be used only for SetupHFSS and SetupAuto, but not for the others, so we can add it to Setup class, and internally check if the Setups are of one of these 2 types. We can do It in another PR.

**Review by gmalinve — APPROVED** [review]

**Review by gmalinve — APPROVED** [review]

**Pull request merged**